docs/tasty-integration.md

```diff
@@ -0,0 +1,28 @@
+# using tasty with ghciwatch
+
+## bubblewrap
+
+Because `ghciwatch` is waiting for lines to come from `ghci`, you
+can end up waiting forever if you don't change the output buffering from `Tasty`. Something like this works:
+
+```haskell
+module TestMain where
+
+import Control.Exception (SomeException, try)
+import System.IO (BufferMode (NoBuffering), hSetBuffering, stdout)
+import Test.Tasty (TestTree, defaultMain, testGroup)
+
+bubblewrap :: IO () -> IO ()
+bubblewrap io = do
+  try io :: IO (Either SomeException ())
+  hSetBuffering stdout NoBuffering
```

Comment:
Is tasty changing the buffering mode? I'm curious if the benefit here is to set the `NoBuffering` mode, or if the benefit is just that we're flushing the buffer